lab/src/Microsoft.ServiceModel.Syndication/tests/SampleTest.cs

```diff
@@ -1,22 +1,132 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Xunit;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
 
 namespace Microsoft.ServiceModel.Syndication.Tests
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Xunit;
+    using Microsoft.ServiceModel.Syndication;
+    using System.Xml;
+    using System.IO;
+
     public static class SampleTest
     {
+                
         [Fact]
-        public static void PassingTest()
+        public static void SyndicationFeed_CreateNewFeed()
         {
+            // *** SETUP *** \\
+            SyndicationFeed sf = new SyndicationFeed("First feed on .net core ever!!", "This is the first feed on .net core ever!", new Uri("https://microsoft.com"));
+
+            XmlWriter xmlw = XmlWriter.Create("FirstFeedEver.xml");
+            Rss20FeedFormatter rssf = new Rss20FeedFormatter(sf);
+
+            // *** EXECUTE *** \\
+            rssf.WriteTo(xmlw);
+        
+            // *** VALIDATE *** \\
+            Assert.True(true);
+
+            // *** CLEANUP *** \\
+            xmlw.Close();
         }
 
         [Fact]
-        public static void FailingTest()
+        public static void SyndicationFeed_Load_Write_Feed()
         {
-            Assert.True(false, "This test is expected to fail");
+            // *** SETUP *** \\\
+            XmlReader xmlr = XmlReader.Create("feedatom.xml");
```

Comment:
Why "feedatom.xml" and "read.xml" live in one test? If they are testing different things, they should live in separate tests.